### PR TITLE
Add more information and structure on standards and certification

### LINKED ISSUE #60: Add interactive overview for standards
Example how it could be: https://docs.gaia-x.eu/framework/ 
Source is [here](https://gitlab.com/gaia-x/technical-committee/framework)

Issue in standards repo: 
https://github.com/SovereignCloudStack/standards/issues/86

### PR DESCRIPTION
**Pull request by maxwolfs**
Aim of this PR is to get users of the docs page an overview and detail information about the certification involving the standards and the decision records. In order to understand it is important to expose the process and explain the concept with concise structure and vocabulary.

* adding better structure, overview with tables and cross references of standards and their metadata such as state, versions, certification and decision records
* implementing content from @alexander-diab and @mbuechse 
* adding first drafts of visualisations 

This first version has a manually edited structure and content. Potential improvement could be to get a lot of this generated via the metadata/frontmatter within the standards.

closes #60 via User Experience 

Preview Deployment: https://docs-staging.scs.community/standards

### CODE REVIEW

File: standards/certification/overview.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Overview
+
+SCS provides a certification framework, which allows to examine the usage of SCS - in essence on cloud environments - with respect to their compliance with certain xcstandards. SCS has a series of different certificate subjects, which are described further below.
```
Comment (anjastrunk) [inline standards/certification/overview.md:3]: What does `xcstandards` mean?

Comment (anjastrunk) [inline standards/certification/overview.md:3]: We decided to rephrase `certificate subject` to `certificate scope`, as decided in https://github.com/SovereignCloudStack/standards/issues/347

Comment (anjastrunk) [inline standards/certification/overview.md:3]:
```suggestion
SCS provides a certification framework, which allows to examine the usage of SCS - in essence on cloud environments - with respect to their compliance with certain xcstandards. SCS has a series of different certificate scopes, which are described further below.
```

File: standards/certification/overview.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+You may think of a certificate as a "software version" - a "guarantee", that two cloud environments, which are certified on the same subject have exactly the same behavior with respect to this subject. This at first provides interoperability, the primary value proposition of SCS. It furthermore allows different operators to share and exchange directly applicable knowledge and experience, leverages external support to interact and intervene with different users and enables integrators to develop and deploy against a single runtime environment irrespective, what the final deployment environment is, as long as it is certified (for this subject).
```
Comment (anjastrunk) [inline standards/certification/overview.md:5]:
```suggestion
You may think of a certificate as a "software version" - a "guarantee", that two cloud environments, which are certified on the same subject have exactly the same behavior with respect to this scope. This at first provides interoperability, the primary value proposition of SCS. It furthermore allows different operators to share and exchange directly applicable knowledge and experience, leverages external support to interact and intervene with different users and enables integrators to develop and deploy against a single runtime environment irrespective, what the final deployment environment is, as long as it is certified (for this subject).
```

File: standards/certification/overview.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+SCS certification subjects ultimately provide the verifable standardization for open source cloud computing stacks - SCS' essential mission.
```
Comment (anjastrunk) [inline standards/certification/overview.md:7]:
```suggestion
SCS certification scopes ultimately provide the verifiable standardization for open source cloud computing stacks - SCS' essential mission.
```

File: standards/certification/get-certified.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# How to get certified
+
+1. (optional) Download the compliance check tool and run it on your environment in order to check that all standards for the certification subject are met.
```
Comment (anjastrunk) [inline standards/certification/get-certified.md:3]:
```suggestion
1. (optional) Download the compliance check tool and run it on your environment in order to check that all standards for the certification scope are met.
```

File: standards/certification/get-certified.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+2. Contact the SCS project team and request to officially certified.
+3. Prepare yourself to give the SCS project team user access to your cloud environment
+4. The SCS project team will run regular tests (the same as the compliance check tool) , in order to check whether all standards compiled in the certification subject are met.
```
Comment (anjastrunk) [inline standards/certification/get-certified.md:6]:
```suggestion
4. The SCS project team will run regular tests (the same as the compliance check tool) , in order to check whether all standards compiled in the certification scope are met.
```

File: standards/certification/get-certified.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+5. If so, then your environment is listed in the "certified clouds" section on this site.
+
+The above workflow applies at the moment to the certification subject SCS-compatible on IaaS. For more complex standards to be met, which may not be automatically and remotely tested.
```
Comment (anjastrunk) [inline standards/certification/get-certified.md:9]:
```suggestion
The above workflow applies at the moment to the certification scope SCS-compatible on IaaS. For more complex standards to be met, which may not be automatically and remotely tested.
```

File: standards/certification/overview.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Overview
+
+SCS provides a certification framework, which allows to examine the usage of SCS - in essence on cloud environments - with respect to their compliance with certain xcstandards. SCS has a series of different certificate scopes, which are described further below.
+
+You may think of a certificate as a "software version" - a "guarantee", that two cloud environments, which are certified on the same subject have exactly the same behavior with respect to this scope. This at first provides interoperability, the primary value proposition of SCS. It furthermore allows different operators to share and exchange directly applicable knowledge and experience, leverages external support to interact and intervene with different users and enables integrators to develop and deploy against a single runtime environment irrespective, what the final deployment environment is, as long as it is certified (for this subject).
```
Comment (anjastrunk) [inline standards/certification/overview.md:5]:
```suggestion
You may think of a certificate as a "software version" - a "guarantee", that two cloud environments, which are certified on the same scope have exactly the same behavior with respect to this scope. This at first provides interoperability, the primary value proposition of SCS. It furthermore allows different operators to share and exchange directly applicable knowledge and experience, leverages external support to interact and intervene with different users and enables integrators to develop and deploy against a single runtime environment irrespective, what the final deployment environment is, as long as it is certified (for this scope).
```

File: standards/certification/overview.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+
+SCS certification scopes ultimately provide the verifiable standardization for open source cloud computing stacks - SCS' essential mission.
+If an entity under examination (like a certain cloud environment) fulfils all required standards for a particular subject a certificate for this particular subject is granted to this entity by a certification assessment body. At the moment the only valid certification assessment body is the SCS project team.
```
Comment (anjastrunk) [inline standards/certification/overview.md:8]:
```suggestion
If an entity under examination (like a certain cloud environment) fulfils all required standards for a particular subject a certificate for this particular scope is granted to this entity by a certification assessment body. At the moment the only valid certification assessment body is the SCS project team.
```

File: standards/certification/overview.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
...
+Like standards certification subjects are discussed, developed and maintained by the technical teams in the community including the active contribution of existing users.
```
Comment (anjastrunk) [inline standards/certification/overview.md:9]:
```suggestion
Like standards certification scopes are discussed, developed and maintained by the technical teams in the community including the active contribution of existing users.
```

File: standards/certification/subjects-versions.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
+# Subjects and Versions
```
Comment (anjastrunk) [inline standards/certification/subjects-versions.md:1]:
```suggestion
# Scope and Versions
```

File: standards/certification/subjects-versions.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+
+The SCS Certification subjects are discrimitated in two dimensions. The first is the technology layer, which is either IaaS (infrastructure) or KaaS (Kubernetes). The second dimension is the level, which refer to the level of sovereignity, which is achieved by the certificate. There are three different levels:
```
Comment (anjastrunk) [inline standards/certification/subjects-versions.md:3]:
```suggestion
The SCS Certification scope are discrimitated in two dimensions. The first is the technology layer, which is either IaaS (infrastructure) or KaaS (Kubernetes). The second dimension is the level, which refer to the level of sovereignty, which is achieved by the certificate. There are three different levels:
```

Comment (anjastrunk) [inline standards/certification/subjects-versions.md:3]: Type? discriminate --> discriminate

Is "discriminate" the right term here? Wouldn't be "split" or "differentiate" fit better?

File: standards/certification/subjects-versions.md [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+
+- SCS compatible
+- SCS open
+- SCS sovereign
+
+Consequently, an SCS user can achieve certificates in six different subjects.
+Hence, an SCS certification subject examines all standards required to achieve a certain level for a certain layer, e.g. the "SCS-compatible" level for the IaaS-layer.
```
Comment (anjastrunk) [inline standards/certification/subjects-versions.md:10]:
```suggestion
Hence, an SCS certification scope examines all standards required to achieve a certain level for a certain layer, e.g. the "SCS-compatible" level for the IaaS-layer.
```

File: standards/certification/subjects-versions.md [hunk 13]
```diff
@@ -0,0 +1,14 @@
...
+
+Like software states, certification subjects are versioned. To achieve the certificate for a subject all standards, which are part of the current version of this subject must be met. A certification subject version has an expiry date after which the next version of this subject becomes valid. An SCS user must fulfil the standards in the actual certification subject version to stay certified. There is a transition period between two versions, in which both versions are valid as shown in the diagram below. It also shows that each subject has its own versioning cycle independent from all other subjects.
```
Comment (anjastrunk) [inline standards/certification/subjects-versions.md:12]:
```suggestion
Like software states, certification scopes are versioned. To achieve the certificate for a scope all standards, which are part of the current version of this scope must be met. A certification scope version has an expiry date after which the next version of this scope becomes valid. An SCS user must fulfill the standards in the actual certification scope version to stay certified. There is a transition period between two versions, in which both versions are valid as shown in the diagram below. It also shows that each scope has its own versioning cycle independent from all other scope.
```

File: standards/certification/subjects-versions.md (hunk 11 above)
Comment (mbuechse) [reply standards/certification/subjects-versions.md]: Likely! I will change it when it revise the text.

File: standards/certification/overview.md (hunk 1 above)
Comment (mbuechse) [reply standards/certification/overview.md]: Most likely a typo. I will change that.

**Comment by mbuechse** [comment]
@anjastrunk I replaced subject by scope and also fixed the other typos you found: https://github.com/SovereignCloudStack/docs/pull/76/files/bd6f388ec97166d90d4b823743f8114d6a7e1de5..080cd241bdb72dce0f7729c8a03f86e0d8127058

Now on to the actual revision...

**Review by mbuechse — CHANGES_REQUESTED** [review]
Hi, I couldn't write my suggestions into these tiny fields in Github, so I pushed a commit instead.

File: standards/certification/image.png
[the remark this answers is not in the document]
Comment (maxwolfs) [reply standards/certification/image.png:1]: I have updated the image!

[the remark this answers is not in the document]
Comment (maxwolfs) [reply standards/certification/image.png:1]: Bildschirmfoto 2023-11-01 um 09 20 29

**Review by mbuechse — APPROVED** [review]

**Pull request merged**